src/tests/ftest/datamover/serial_large_posix.yaml

```diff
@@ -49,3 +49,4 @@ ddeserialize:
     np: 16
 dfuse:
   disable_caching: true
+  local_flock: true
```

Comment:
I'm curious about how tests were chosen to use `local_flock: true`? Was it just arbitrary to get some coverage?
FWIW this test serializes a container into dfuse in an HDF5 format.

Reply:
@daltonbohning Actually, I ran all CI tests without "local_flock: true" at the beginning. Three tests failed. I discussed with Mohamad whether we could set "local_flock: true" for them.

Reply:
So we set `local_flock: true` in certain tests because they fail without it? In a production environment, how would a user know to set this, since it's not the default?

Reply:
we do not support flock. enabling it by default through fuse gives the false sense that it is supported. so if one does flock across multiple nodes, both nodes can acquire the lock at the same time and get granted which can cause data corruption.
The idea is to disable this behavior since this is not what we want. so in production users would get errors when their apps use flock, as it should be.
This flag is to allow the use of flock in certain situation where the users know what they are doing, mainly running on a single node with a single dfuse instance. In that case, using flock is safe.

Reply:
Thanks for the explanation

Reply:
Would it be reasonable to only allow flock to be enabled if the container has been opened with the `DAOS_COO_EX` flag set?